book/make_docs.nu

```diff
@@ -89,16 +91,24 @@ def get-doc [command] {
 
   let parameters = if $no_param { "" } else { $"## Parameters(char nl)(char nl)($params)(char nl)(char nl)" }
 
-  let extra_usage = if $command.extra_usage == "" { "" } else {
-    # It's a little ugly to encode the extra usage in a code block,
-    # but otherwise Vuepress's Markdown engine makes everything go haywire
-    # (the `ansi` command is a good example).
-    # TODO: find a better way to display plain text with minimal formatting
-    $"## Notes
-```text
-($command.extra_usage)
-```
-"
+  let ex = $command.extra_usage
+  # Certain commands' extra_usage is wrapped in code block markup to prevent their code from
+  # being interpreted as markdown. This is strictly hard-coded for now.
+  let extra_usage = if $ex == "" { "" } else if $command.command in ['def-env' 'export def-env' 'as-date' 'as-datetime' ansi] {
```

Comment:
Can we define `['def-env' 'export def-env' 'as-date' 'as-datetime' ansi]` as a const variable?